modules/default.nix

```
@@ -123,6 +123,12 @@ in
           binaries needed for flashing/fusing Jetson SOMs.
         '';
       };
+
+      console.enable = mkOption {
+        default = true;
```

```suggestion
        default = config.console.enable;
        defaultText = "config.console.enable";
```
 There exists an option upstream that might be nice to mirror that value here.